corehq/form_processor/parsers/form.py

```diff
@@ -96,13 +98,14 @@ def _create_new_xform(domain, instance_xml, attachments=None, auth_context=None)
     interface = FormProcessorInterface(domain)
 
     assert attachments is not None
-    form_data = convert_xform_to_json(instance_xml)
-    if not form_data.get('@xmlns'):
+    if not instance_json:
```

Comment:
nit: Not sure if `instance_json` could ever be an empty dict, but it seems best to be strict and only convert XML if it is null.
```suggestion
    if instance_json is not None:
```

Reply:
963cb4275a469669a08dd7ce1056e3f99d39057e

Reply:
oh lol 0aa748286c0d4c56fedf8c7266d1af309cbbebf5